vllm/worker/model_runner.py

```diff
@@ -248,15 +227,26 @@ def __init__(self, request_id: str, seq_ids: List[int],
 
             self.lora_index_mapping = [[] for _ in range(self.n_seqs)]
             self.lora_prompt_mapping = [[] for _ in range(self.n_seqs)]
-            self.lora_requests = set()
-
-            self.prompt_adapter_index_mapping = []
-            self.prompt_adapter_prompt_mapping = []
 
     def __init__(self,
                  runner: "GPUModelRunnerBase",
                  finished_requests_ids: Optional[List[str]] = None):
         super().__init__()
+        # Compute functions for each sequence in a sequence group.
+        # WARNING: The order of the functions matters!
+        self.per_seq_compute_fns = (
```

Comment:
let's make them lists instead of tuples

Reply:
Interesting. My impression is tuple is more efficient for iteration because it's immutable, but would like to know your opinion

Reply:
making them mutable will let end user modify them easier, but it's not a big deal either way.